google/resource_container_node_pool_test.go

```diff
@@ -92,6 +92,24 @@ func TestAccContainerNodePool_withNodeConfig(t *testing.T) {
 	})
 }
 
+func TestAccContainerNodePool_withGPU(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerNodePoolDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccContainerNodePool_withGPU(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckContainerNodePoolMatches("google_container_node_pool.np_with_gpu"),
+				),
+			},
```

Comment:
Looks great - can you stick an ImportState step on there so we know that accelerated node pools can be imported?

Reply:
It would be the only ImportState in this file - why introduce it for just this case?

Reply:
We've talked to Hashicorp about best practices recently, and so it's a new thing we're doing with new tests.